files/zh-cn/glossary/ssr/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 服务器端渲染（SSR）
+slug: Glossary/SSR
+l10n:
+  sourceCommit: d864c4fa589174ddd906bb679576d2265a20fad1
+---
+
+{{GlossarySidebar}}
+
+**服务器端渲染**（SSR）指的是在服务器上生成 HTML 内容并将其发送到客户端的做法。SSR 与{{glossary("CSR", "客户端渲染")}}相对，后者由客户端使用 JavaScript 生成 HTML 内容。这两种技术并不互斥，可以在同一个应用中结合使用。
+
+{{glossary("SSG", "静态网站")}}可以被视为 SSR 的一种（并且可以使用 SSR 的基础设施生成），但两者有细微差别。静态网站的内容是在构建时生成的，而不是在请求时生成。静态网站通常无需部署在服务器上，可以通过 {{glossary("CDN")}} 进行分发。
+
+SSR/CSR 的区别在于动态内容的网站更为重要，例如需要实时更新或用户特定内容的站点。在这些情况下，服务器会为每个请求动态生成 HTML 内容，因为预先生成所有可能页面是不现实的。HTML 文件包含几乎完整的页面内容，JavaScript 资源仅用于实现交互。
```

Comment:
SSR/CSR 的区别在于xxx更为重要？读不通，读不明白

Reply:
The SSR/CSR distinction is more meaningful for sites with dynamic content, for example live-updating or user-specific content. In these cases, for every request, the server generates the HTML content on-the-fly because it is unrealistic to pregenerate every possible page. The HTML file contains near-complete page content, and any JavaScript asset is only to enable interactivity.

主次翻译反了，不好意思……

我感觉可以翻译成这个样子：

```suggestion
对于那些使用了动态内容的站点（比如实时更新，或者根据用户客制化内容的站点）而言，区分 SSR/CSR 更为重要。在这些情况下，服务器要为每个请求动态生成 HTML 内容，因为预先穷举生成所有可能的页面是不现实的。这些生成的 HTML 文件包含近乎完整的内容，而 JavaScript 资源则仅用于实现交互功能。
```